tests/modules/test_securitydata.py

```diff
@@ -919,9 +925,11 @@ def test_stream_file_by_md5_raises_file_not_found_error_when_search_returns_empt
         file_event_search.text = "{}"
         file_event_client = mocker.MagicMock(spec=FileEventClient)
         file_event_client.search.return_value = file_event_search
-        with pytest.raises(Py42Error):
```

Comment:
This test name hasn't changed in this PR, still says `file_not_found_error`

Reply:
thanks, done.